### ISSUE TITLE
Write API description for dailyQuote

### ISSUE THREAD

**Issue by zudiay** [bounswe/2021SpringGroup5#178]
I have already provided a readme that explains the functionality of the app and how to run it [here](https://github.com/bounswe/2021SpringGroup5/tree/master/practice-app/group5_api/dailyQuote). I will provided a detailed API documentation that explains the endpoints, methods, responses and functions.

**Comment by zudiay** [comment]
I completed writing the API documentation. I wrote a single & detailed documentation where 
- I explained the app briefly
- I explained the code, classes, serializers, views and functions in detail
- I documented the API endpoints, the parameters, expected responses and some example responses. 
- I documented how to run the app and how to use the user interface
- I added some screenshots of the app. 
- I provided the unit tests. 
I am not putting the documentation to the repository now, just in case I make some adjustments, changes in the code and I need to update the documentation.
